tests/msc2716_test.go

```diff
@@ -527,11 +547,6 @@ func TestImportHistoricalMessages(t *testing.T) {
 			})
```

Comment:
TODO: Use `validateBatchSendRes` here (https://github.com/matrix-org/complement/pull/219 was merged in)